env/build.go

```diff
@@ -17,12 +19,29 @@ var BuildEnvIncludelist = []string{
 	"no_proxy",
 }
 
+// List of environment variables to be included for
+// particular platform API versions
```

Comment:
```suggestion
// asset packages
```

I assume if we need to update the build env for future platform api versions we'd create a new array and call it something other than `BuildEnvIncludeAssets`.